packages/terra-horizontal-divider/tests/nightwatch/horizontal-divider-spec.js

```diff
@@ -0,0 +1,19 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+// eslint-disable-next-line import/no-extraneous-dependencies
+const screenshot = require('terra-toolkit').screenshot;
+
+module.exports = {
+  before: (browser, done) => {
+    browser.resizeWindow(browser.globals.width, browser.globals.height, done);
+  },
+
+  afterEach: (browser, done) => {
+    screenshot(browser, 'terra-horizontal-divider', done);
+  },
+
+  'Displays a default horizontal-divider': (browser) => {
+    browser
+      .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/horizontal-divider-tests/default`)
+      .assert.elementPresent('.horizontal-divider');
```

Comment:
Use the nightwatch expect syntax, not assert syntax. http://nightwatchjs.org/api/#expect-api

Reply:
Updated https://github.com/cerner/terra-core/commit/b6a950329f831a533a9f38d54427d9c17a5e17e6#diff-342e0ebf47dd992b3c58fb3777d1ac69R8